src/erp12/cbgp_lite/lang/lib.clj

```diff
@@ -401,10 +496,20 @@
    ;'min-double          (binary-transform DOUBLE)
    ;'max-int             (binary-transform INT)
    ;'max-double          (binary-transform DOUBLE)
+   'sqrt               (unary-transform DOUBLE)
    `sin                (unary-transform DOUBLE)
    `cos                (unary-transform DOUBLE)
    `tan                (unary-transform DOUBLE)
-   ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+   'asin                (unary-transform DOUBLE)
+   'acos                (unary-transform DOUBLE)
```

Comment:
I think these can be \`asin and \`acos  (instead of 'asin and 'acos) and then you won't need an entry in `dealiases`.